tests/test_2023_10_31.py

```diff
@@ -0,0 +1,657 @@
+    assert contracts.usdt_token.balanceOf(stranger) == usdt_balance_before + 2_000_000 * 10**6
+
+    # MATIC
+    # Check token that is not supported.
+    MATIC_TOKEN = "0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0"
+    MATIC_HOLDER = "0x5e3Ef299fDDf15eAa0432E6e66473ace8c13D908"
+    matic_token = interface.ERC20(MATIC_TOKEN)
+    matic_token.transfer(agent.address, 1, { 'from': MATIC_HOLDER })
+    with reverts("APP_AUTH_FAILED"):
+        contracts.finance.newImmediatePayment(
+            matic_token,
+            stranger,
+            1,
+            "MATIC transfer",
+            {"from": evmscriptexecutor},
+        )
+
+    ## todo: uncomment tests
+    # 1. Remove RCC DAI top up EVM script factory (old ver) 0x84f74733ede9bFD53c1B3Ea96338867C94EC313e from Easy Track
+    # assert rcc_dai_topup_factory_old not in updated_factories_list
+    # 2. Remove PML DAI top up EVM script factory (old ver) 0x4E6D3A5023A38cE2C4c5456d3760357fD93A22cD from Easy Track
+    # assert pml_dai_topup_factory_old not in updated_factories_list
+    # 3. Remove ATC DAI top up EVM script factory (old ver) 0x67Fb97ABB9035E2e93A7e3761a0d0571c5d7CD07 from Easy Track
+    # assert atc_dai_topup_factory_old not in updated_factories_list
+
+    # 4. Add RCC stable top up EVM script factory 0x84f74733ede9bFD53c1B3Ea96338867C94EC313e to Easy Track
+    assert rcc_stable_topup_factory in updated_factories_list
+    create_and_enact_payment_motion(
+        contracts.easy_track,
+        rcc_trusted_caller_and_recepient,
+        rcc_stable_topup_factory,
+        contracts.dai_token,
+        [rcc_trusted_caller_and_recepient],
+        [10 * 10**18],
+        stranger,
+    )
+    check_add_and_remove_recipient_with_voting(rcc_stable_registry, helpers, LDO_HOLDER_ADDRESS_FOR_TESTS, contracts.voting)
+
+    # 5. Add PML stable top up EVM script factory 0x4E6D3A5023A38cE2C4c5456d3760357fD93A22cD to Easy Track
+    assert pml_stable_topup_factory in updated_factories_list
+    create_and_enact_payment_motion(
+        contracts.easy_track,
+        pml_trusted_caller_and_recepient,
+        pml_stable_topup_factory,
+        contracts.dai_token,
+        [pml_trusted_caller_and_recepient],
+        [10 * 10**18],
+        stranger,
+    )
+    check_add_and_remove_recipient_with_voting(pml_stable_registry, helpers, LDO_HOLDER_ADDRESS_FOR_TESTS, contracts.voting)
+
+    # 6. Add ATC stable top up EVM script factory 0x67Fb97ABB9035E2e93A7e3761a0d0571c5d7CD07 to Easy Track
+    assert atc_stable_topup_factory in updated_factories_list
+    create_and_enact_payment_motion(
+        contracts.easy_track,
+        atc_trusted_caller_and_recepient,
+        atc_stable_topup_factory,
+        contracts.dai_token,
+        [atc_trusted_caller_and_recepient],
+        [10 * 10**18],
+        stranger,
+    )
+    check_add_and_remove_recipient_with_voting(atc_stable_registry, helpers, LDO_HOLDER_ADDRESS_FOR_TESTS, contracts.voting)
+
```

Comment:
Could you please add tests for stETH payments?
1. agent’s balance has decreased by the expected number. 
2. multisig balances increased by expected amounts.
3. agent delta = sum of multisig deltas.